lookout/style/typos/corrector.py

```diff
@@ -136,7 +136,8 @@ def suggest(self, data: pandas.DataFrame, candidates: str = None,
         :param save_candidates_file: Path to file to save candidates to (.csv.xz).
         :param n_candidates: Number of most probable candidates to return.
         :param return_all: False to return suggestions only for corrected tokens.
-        :return: Dictionary {id : [[candidate, correctness_proba]]}, candidates are sorted \
+        :param save_candidates_file: Path to file to save candidates to (.pkl).
+        :return: Dictionary `{id : [[candidate, correctness_proba]]}`, candidates are sorted \
```

Comment:
`[(candidate, correctness_proba), …]` will better reflect the fact that each value is a list of tuples (and this should probably be updated everywhere here and in ranker)